pkg/report/writer.go

```diff
@@ -57,6 +57,7 @@ type Result struct {
 	Vulnerabilities   []types.DetectedVulnerability    `json:"Vulnerabilities,omitempty"`
 	MisconfSummary    *MisconfSummary                  `json:"MisconfSummary,omitempty"`
 	Misconfigurations []types.DetectedMisconfiguration `json:"Misconfigurations,omitempty"`
+	Size              int64                            `json:"Size,omitempty"`
```

Comment:
Put it into report.Metadata
https://github.com/aquasecurity/trivy/blob/5e2ca95c723c55311ae4354554bbadc1cab829e4/pkg/report/writer.go#L30-L38